goosebit/api/devices/device/routes.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from fastapi import APIRouter, Depends, Security
+from fastapi.requests import Request
+
+from goosebit.api.devices.device.responses import DeviceLogResponse, DeviceResponse
+from goosebit.auth import validate_user_permissions
+from goosebit.permissions import Permissions
+from goosebit.updater.manager import UpdateManager, get_update_manager
+
+router = APIRouter(prefix="/{dev_id}")
+
+
+@router.get(
+    "",
```

Comment:
Thanks for using `""` instead of `"/"`. The latter can cause redirect loops (at least with our deployment on AWS).

Reply:
Yeah I noticed that it seems to cause some issues, the only place it is required is at the root, but any router under a router can use `""`.